scripts/populate_store/upload_bundles.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import json
+import requests
+
+
+def upload_structure_definitions(bundle_files):
+
+    resource_types = ["StructureDefinition", "ConceptMap", "CodeSystem"]
```

Comment:
why do we force to use these resource types ? this script could be used to upload all kinds of FHIR bundles don't you think ?

Reply:
We could, it's just that there are many things in the resources.json bundle we got from hl7.org, didn't know if we wanted all of them